### PR TITLE
Side effect implication notification

### PR DESCRIPTION
**Pull request by Dunfluff**

### CODE REVIEW

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 1]
```diff
@@ -923,6 +923,26 @@ The product's default behavior of using a cost-based IDP search algorithm when c
 For more information, see link:https://neo4j.com/docs/cypher-manual/current/query-tuning/query-options/#cypher-planner[Cypher manual -> Cypher planner].
 ====
 
+.Using a subquery expression within a mutating statement has implications for its side-effect visibility.
```
Comment (renetapopova) [inline modules/ROOT/pages/notifications/all-notifications.adoc:926]:
```suggestion
.Using a subquery expression within a mutating statement has implications for its side-effect visibility
```

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 2]
```diff
@@ -923,6 +923,26 @@ The product's default behavior of using a cost-based IDP search algorithm when c
...
+====
+Query::
++
+[source,cypher]
+----
+CREATE ( { prop : COUNT { () } } )
+----
+Description of the returned code::
+The semantics of this statement may change in later versions.
+Suggestions for improvement::
```
Comment (renetapopova) [inline modules/ROOT/pages/notifications/all-notifications.adoc:936]: Maybe skip a line just for readability.
```suggestion

Suggestions for improvement::
```

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 3]
```diff
@@ -923,6 +923,26 @@ The product's default behavior of using a cost-based IDP search algorithm when c
...
+Extract the subquery expression into a preceding WITH and/or wrap the mutating statement into a CALL subquery. For further explanation please contact support.
```
Comment (renetapopova) [inline modules/ROOT/pages/notifications/all-notifications.adoc:937]:
```suggestion
Extract the subquery expression into a preceding `WITH` and/or wrap the mutating statement into a `CALL` subquery. 
For further explanation, please contact support.
```

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 4]
```diff
@@ -923,6 +923,26 @@ The product's default behavior of using a cost-based IDP search algorithm when c
...
 
+.Using a subquery expression within a mutating statement has implications for its side-effect visibility.
+====
+Query::
++
+[source,cypher]
+----
+CREATE ( { prop : COUNT { () } } )
+----
+Description of the returned code::
```
Comment (renetapopova) [inline modules/ROOT/pages/notifications/all-notifications.adoc:934]: Maybe skip a line just for readability.
```suggestion

Description of the returned code::
```

**Review by renetapopova — COMMENTED** [review]
Some editorial suggestions.

**Review by renetapopova — APPROVED** [review]
Looks good to me! Thanks.

**Comment by Dunfluff** [comment]
image

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 5]
```diff
@@ -923,6 +923,28 @@ The product's default behavior of using a cost-based IDP search algorithm when c
 For more information, see link:https://neo4j.com/docs/cypher-manual/current/query-tuning/query-options/#cypher-planner[Cypher manual -> Cypher planner].
 ====
 
+.Using a subquery expression within a mutating statement has implications for its side-effect visibility
```
Comment (Lojjs) [inline modules/ROOT/pages/notifications/all-notifications.adoc:926]: Now you have added this under deprecation. You should instead add it under a new section with the new code, similar to what Jack did here: https://github.com/neo4j/docs-status-codes/pull/75/files

**Comment by Dunfluff** [comment]
Updated status code documentation
image

**Review by Lojjs — APPROVED** [review]

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 6]
```diff
@@ -923,6 +923,52 @@ The product's default behavior of using a cost-based IDP search algorithm when c
...
+[#_neo_clientnotification_statement_sideeffectvisibility]
+=== SideEffectVisibility
+
+.Notification details
+[cols="<1s,<4"]
+|===
+|Code
+m|Neo.ClientNotification.Statement.SideEffectVisibility
+|Title
+a| Using a subquery expression within a mutating statement has implications for its side-effect visibility
```
Comment (renetapopova) [inline modules/ROOT/pages/notifications/all-notifications.adoc:935]:
```suggestion
a| A subquery expression within a mutating statement has implications for its side-effect visibility.
```

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 7]
```diff
@@ -923,6 +923,52 @@ The product's default behavior of using a cost-based IDP search algorithm when c
...
+
+Suggestions for improvement::
+Extract the subquery expression into a preceding `WITH`
++
+[source,cypher]
+----
+WITH COUNT {()} AS i
+CREATE ( { prop : i } )
+----
+and potentially wrap the mutating statement into a `CALL` subquery if changes from previous executions should be observable
```
Comment (renetapopova) [inline modules/ROOT/pages/notifications/all-notifications.adoc:962]:
```suggestion
+
and potentially wrap the mutating statement into a `CALL` subquery if changes from previous executions should be observable:
+
```

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 8]
```diff
@@ -923,6 +923,52 @@ The product's default behavior of using a cost-based IDP search algorithm when c
...
+[source,cypher]
+----
+CREATE ( { prop : COUNT { () } } )
+----
+
+Description of the returned code::
+The semantics of this statement may change in later versions.
+
+Suggestions for improvement::
+Extract the subquery expression into a preceding `WITH`
```
Comment (renetapopova) [inline modules/ROOT/pages/notifications/all-notifications.adoc:955]:
```suggestion
Extract the subquery expression into a preceding `WITH`:
```

**Review by renetapopova — COMMENTED** [review]
Some editorial suggestions.

**Comment by renetapopova** [comment]
Hey @Dunfluff, which version is this PR applicable to?

**Pull request merged**